### DISCUSSION TITLE
spacy for medical terms

### DISCUSSION THREAD

**Discussion by JGz4PS8DfvVb53** [explosion/spaCy#12207] [Language Support]
Which spacy library works best for similarities of german medical terms.
e.g. to show this synonyms: Cor (latin) = Herz (german) = heart (english)

which of the 3 spacy libraries works best for that:
- medspaCy
- scispaCy
- Kindred

**Comment by polm** [comment]
Sorry, we're not familiar enough with all third party packages to help with a question like that. You might have more luck asking the respective packages, though it's probably better to try them out for yourself. 

That said, looking at the packages briefly, it looks like none of them have any features for cross-lingual comparisons. The first thing that comes to mind with a simlar feature is WordNet, but it's not designed for specialist language.

**Reply by JGz4PS8DfvVb53** [reply]
thank you for the fast answer!
Seems to be more difficult than expected to find a library for german medical terms.

**Comment by JGz4PS8DfvVb53** [comment]
Has possibly somebody experience with BioWordVed whether it covers german terms as well? A library with 27GB.

https://github.com/ncbi-nlp/BioWordVec
